### DISCUSSION TITLE
MacOS Sonoma (DisplayCal Python) + Blackmagic UltraStudio (DaVinci Resolve)

### DISCUSSION THREAD

**Discussion by Bastizor** [eoyilmaz/displaycal-py3#307] [General]
Hi,

As you know, the native MacOS app doesn’t work since MacOS Sonoma has been released.
So I tried using your Python 3 version.

As far as the DisplayCal Python works properly with external monitor, it doesn’t work within Resolve with the UltraStudio I/O device.

It doesn’t work when using DaVinci Resolve with an i/o device, such as the Blackmagic UltraStudio Mini 4K : the color patches are not sent to the i/o device, so not displayed on the monitor plugged into it. The IP connection is working fine though. The calibration fails at the very first steps. I only see a grey patch and then the measurements give an error.

Do you have any clue if this is something you could fix or implement in the near future for the version of DisplayCal that you have ported to Python 3?

This topic is also discussed here : [https://hub.displaycal.net/forums/topic/macos-sonoma-displaycal-python-blackmagic-ultrasutdio-davinci-resolve/#post-140006]

Thank you

**Comment by eoyilmaz** [comment]
This has been resolved with DisplayCAL 3.9.12
